### ISSUE TITLE
Compound Verb Capitalization

### ISSUE THREAD

**Issue by natescherer** [MicrosoftDocs/PowerShell-Docs#3677]
Can someone clarify proper capitalization for compound verbs like 'convertfrom' and 'convertto'?

The general verb naming rules say this: 'Use Pascal casing for verbs. In Pascal casing the initial letter of each word is capitalized, such as "ForEach".'

The list of verbs has 'convertfrom' and 'convertto' styled as 'Convertfrom' and 'Convertto'.

I took the primary list of verbs as correct when developing my own modules, but I realized that ConvertFrom-Csv (built into PowerShell) used the Pascal casing, so I was confused.

Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 51a81c73-9edf-2815-268d-a7b294fe6bd3
* Version Independent ID: eb63eebc-a28c-be84-fd47-e7573f09e0b0
* Content: [Approved Verbs for PowerShell Commands](https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands)
* Content Source: [developer/cmdlet/approved-verbs-for-windows-powershell-commands.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/developer/cmdlet/approved-verbs-for-windows-powershell-commands.md)
* Product: **powershell**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

**Comment by sdwheeler** [comment]
@natescherer 
```
PS > get-verb convert*

Verb        AliasPrefix Group Description
----        ----------- ----- -----------
Convert     cv          Data  Changes the data from one representation to another when the cmdlet supports bidirecti...
ConvertFrom cf          Data  Converts one primary type of input (the cmdlet noun indicates the input) to one or mor...
ConvertTo   ct          Data  Converts from one or more types of input to a primary output type (the cmdlet noun ind...
```

**Comment by natescherer** [comment]
Perhaps I didn’t phrase it clearly enough, but the reason why I made this issue here is because the documentation conflicts with itself, I wasn’t looking for support. If you look at to the Common Verbs section in the document you’ll see they aren’t capitalized that way, despite the General Rules section earlier in the document saying that they should be.

**Comment by sdwheeler** [comment]
The documentation needs updating. It is inconsistent in many places. We are working on updates but there is a lot of work to do.

**Comment by doctordns** [comment]
Fixed in #3681
